src/sentry/integrations/bitbucket/client.py

```diff
@@ -0,0 +1,203 @@
+from __future__ import absolute_import
+
+import datetime
+import jwt
+
+from unidiff import PatchSet
+
+from six.moves.urllib.parse import urlparse
+
+from sentry.utils.http import absolute_uri
+from sentry.integrations.atlassian_connect import get_query_hash
+from sentry.integrations.client import ApiClient
+
+BITBUCKET_KEY = '%s.bitbucket' % urlparse(absolute_uri()).hostname
+
+
+class BitbucketAPIPath(object):
+    """
+    All UUID's must be surrounded by culybraces.
+    repo_slug - repository slug or UUID
+    username - username or UUID
+    """
+
+    issue = u'/2.0/repositories/{username}/{repo_slug}/issues/{issue_id}'
+    issues = u'/2.0/repositories/{username}/{repo_slug}/issues'
+    issue_comments = u'/2.0/repositories/{username}/{repo_slug}/issues/{issue_id}/comments'
+
+    repository = u'/2.0/repositories/{username}/{repo_slug}'
+    repository_commits = u'/2.0/repositories/{username}/{repo_slug}/commits/{revision}'
+    repository_diff = u'/2.0/repositories/{username}/{repo_slug}/diff/{spec}'
+    repository_hook = u'/2.0/repositories/{username}/{repo_slug}/hooks/{uid}'
+    repository_hooks = u'/2.0/repositories/{username}/{repo_slug}/hooks'
+
+
+class BitbucketAPI(ApiClient):
+
+    def __init__(self, base_url, shared_secret, subject):
+        # subject is probably the clientKey
+        super(BitbucketAPI, self).__init__(verify_ssl=False)
```

Comment:
do you need the `verify_ssl` to be false for this to work? because it looks like the old bitbucket client didn't need to

Reply:
@macqueen good catch! thank you~!